README.md

```diff
@@ -2,36 +2,50 @@
 
 ** Warning: This is not production ready! **
 
-Instrumental is an [application monitoring platform](https://instrumentalapp.com) built for developers who want a better understanding of their production software. Powerful tools, like the [Instrumental Query Language](https://instrumentalapp.com/docs/query-language), combined with an exploration-focused interface allow you to get real answers to complex questions, in real-time.
+Instrumental is an [application monitoring platform](https://instrumentalapp.com/) built for developers who want a better understanding of their production software. Powerful tools, like the [Instrumental Query Language](https://instrumentalapp.com/docs/query-language), combined with an exploration-focused interface allow you to get real answers to complex questions, in real-time.
+
+**Instrumental System Daemon** (ISD for short) is a server agent that provides [system monitoring](#system-metrics) and [service monitoring](#service-metrics). It's fast, reliable, runs on *nix and Windows, is [simple to configure](conf/instrumental.toml) and deploy, and has a small memory footprint.
 
-ISD provides [server monitoring](https://instrumentalapp.com/docs/server-monitoring) through the `instrumentald` daemon. It provides high-data reliability at high scale.
 
 ## Installation
-`instrumentald` is currently officially supported on 32-bit and 64-bit Linux, Windows systems and Mac OS X. There are prebuilt packages available for Debian, Ubuntu, RHEL and Win32 systems.
+`instrumentald` is supported on 32-bit and 64-bit Linux, Windows and OSX/macOS. There are prebuilt packages available for Debian, Ubuntu, RHEL and Win32 systems.
```

Comment:
As of the soon to be released next version, it's called macOS. Old versions will likely still be called OSX. This is future proofing. If you want to reduce it, we should make it macOS only.